internal/driver/driver.go

```diff
@@ -70,22 +83,26 @@ func (d *Driver) Initialize(lc logger.LoggingClient, asyncCh chan<- *sdkModel.As
 	d.config = camConfig
 
 	deviceService := sdk.RunningService()
+	d.svc = &DeviceSDKService{
+		DeviceService: deviceService,
+		lc:            lc,
+	}
 
-	for _, device := range deviceService.Devices() {
+	for _, dev := range deviceService.Devices() {
```

Comment:
why shorten name when `device` is a better name for readability?

Reply:
I can change them all back to `device`, but maybe we should consider using an import alias for that package?

Reply:
Yes, prefer an alias for the package in these cases.